easybuild/easyblocks/t/tensorflow.py

```diff
@@ -142,11 +170,16 @@ def configure_step(self):
         tensorrt_root = get_software_root('TensorRT')
         nccl_root = get_software_root('NCCL')
 
-        use_mpi = self.toolchain.options.get('usempi', False)
+        mpi_home = ''
+        if use_mpi:
+            impi_root = get_software_root('impi')
```

Comment:
@akesandgren Since you're using this in two places in `configure_step`, why not combine this with the exact same check above to put the `mpiicc` wrapper in place?

```python
use_mpi = self.toolchain.options.get('usempi', False)
impi_root = get_software_root('impi')
mpi_home = ''
if use_mpi and impi_root:
    # put wrappers for Intel MPI compiler wrappers in place
    ...
    # set correct value for MPI_HOME
    mpi_home = os.path.join(impi_root, 'intel64')

Reply:
Done